utils/analysis/helpers.go

```diff
@@ -373,7 +372,17 @@ func NewAnalysisRunFromUnstructured(obj *unstructured.Unstructured, templateArgs
 		return nil, err
 	}
 
+	// Remove resourceVersion if exists
+	_, found, err := unstructured.NestedString(obj.Object, "metadata", "resourceVersion")
```

Comment:
Does this fix imply we never had `--from in-cluster-analysis-template` working properly?

Reply:
yeah, the unit test failed to catch that because of the fake client.